dist/PublicLab.Editor.js

```diff
@@ -22475,7 +22475,7 @@ module.exports = function(textarea, _editor, _module) {
     );
     $(".wk-commands, .wk-switchboard").addClass("btn-group");
     $(".wk-commands button, .wk-switchboard button").addClass(
-        "btn btn-light"
+        "btn btn-outline-secondary"
```

Comment:
Hi @NARUDESIGNS, is this change related to this PR?

Everything else looks great btw. Great job!

Reply:
@TildaDares this change happens automatically when I work on src/modules/PublicLab.RichTextModule.Table.js and I don't know why. Do I need to take it out manually?

Reply:
@NARUDESIGNS Undo the changes and then do a push. Let's see if that works.

Reply:
I should undo the changes just for dist/Publiclab.Editor.js right?

Reply:
> @TildaDares this change happens automatically when I work on src/modules/PublicLab.RichTextModule.Table.js and I don't know why. Do I need to take it out manually?

Also just to update you, this automatic change (addition of "btn btn-outline-secondary" property) in the dist/Publiclab.Editor.js happens in my local env just before I push to remote repo this is why I was asking you if I need to manually remove the change.
@TildaDares

Reply:
Hi @NARUDESIGNS, I'm not sure why but that change is added every time you run `grunt build`.

Reply:
The problem is with this commit https://github.com/publiclab/PublicLab.Editor/commit/b1cb34626b1c3acf8a9d87f12c4ee4830d5f8471. `grunt build` was supposed to be run after adding the change so every time anyone else runs `grunt build`, it adds that change to the `dist` folder.

Leave the change in one of your PRs and remove it from the other. All you have to do is undo just that line of change after running  `grunt build`.

Reply:
Ok, thank you very much. I have removed it after running `grunt build` for PR https://github.com/publiclab/PublicLab.Editor/pull/758 while the change remains here on PR https://github.com/publiclab/PublicLab.Editor/pull/757 just like you advised.
Please review, thank you so much for your time @TildaDares